### PR TITLE
About page - added team section

### PR DESCRIPTION
**Pull request by muzammalrahim**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate_web/templates/about.html [hunk 1]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                <div class="weblate-team-content">
+                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                     <div class="content-p">
+                       <p>MICHAL CIHAR</p>
+                       <br>
+                       <t>Master of the code & head of custom or large scale solutions.</t>
+                    </div>
+                </div>
+                <div class="weblate-team-content">
+                    <img src="{% static "img/donald.png" %}" alt="Quotes" />
```
Comment (nijel) [inline weblate_web/templates/about.html:58]:
```suggestion
                    <img src="https://hosted.weblate.org/avatar/80/orangesunny.png" alt="Benjamin Alan Jamie" />
```

File: weblate_web/templates/about.html [hunk 2]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
                 </div>
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
```
Comment (nijel) [inline weblate_web/templates/about.html:50]:
```suggestion
                   <img src="https://hosted.weblate.org/avatar/80/nijel.png" alt="Michal Čihař" />
```

File: weblate_web/templates/about.html [hunk 3]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                <div class="weblate-team-content">
+                    <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                    <div class="content-p">
+                        <p>BENJAMIN ALAN JAMIE</p>
+                        <br>
+                        <t>Community manager & support guru helping with smooth workflows.</t>
+                    </div>
+                    </div>
+                    <div class="weblate-team-content">
+                        <img src="{% static "img/donald.png" %}" alt="Quotes" />
```
Comment (nijel) [inline weblate_web/templates/about.html:66]:
```suggestion
                        <img src="https://hosted.weblate.org/avatar/80/vitavalka.png" alt="Víťa Válka" />
```

File: weblate_web/templates/about.html [hunk 4]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                        <div class="content-p">
+                            <p>VITA VALKA</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:68]:
```suggestion
                            <p>Víťa Válka</p>
```

File: weblate_web/templates/about.html [hunk 5]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                     <div class="content-p">
+                       <p>MICHAL CIHAR</p>
+                       <br>
+                       <t>Master of the code & head of custom or large scale solutions.</t>
+                    </div>
+                </div>
+                <div class="weblate-team-content">
+                    <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                    <div class="content-p">
+                        <p>BENJAMIN ALAN JAMIE</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:60]:
```suggestion
                        <p>Benjamin Alan Jamie</p>
```

File: weblate_web/templates/about.html [hunk 6]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
                 </div>
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                     <div class="content-p">
+                       <p>MICHAL CIHAR</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:52]:
```suggestion
                       <p>Michal Čihař</p>
```

Please add CSS for uppercasing the stirng.

File: weblate_web/templates/about.html [hunk 7]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                       <br>
+                       <t>Master of the code & head of custom or large scale solutions.</t>
```
Comment (nijel) [inline weblate_web/templates/about.html:54]: Please make the text (and all others translatable).

**Comment by nijel** [comment]
@orangesunny Please review the texts here.

File: weblate_web/templates/about.html (hunk 1 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: It's been added.

File: weblate_web/templates/about.html (hunk 2 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Added.

File: weblate_web/templates/about.html (hunk 3 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Added.

File: weblate_web/templates/about.html (hunk 4 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: It's been changed.

File: weblate_web/templates/about.html (hunk 5 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: It's been changed.

File: weblate_web/templates/about.html (hunk 6 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Upercasing css has been added in both style.css and style-rtl.css

File: weblate_web/templates/about.html (hunk 7 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: The "trans" block has been added to the texts.

File: weblate_web/templates/about.html [hunk 8]
```diff
@@ -47,27 +47,27 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
-                        <t>Community manager & support guru helping with smooth workflows.</t>
+                        <t>{% trans "Community manager & support guru helping with smooth workflows." %}</t>
                     </div>
                     </div>
                     <div class="weblate-team-content">
-                        <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                        <img src="https://hosted.weblate.org/avatar/80/vitavalka.png" alt="Víťa Válka" />
                         <div class="content-p">
-                            <p>VITA VALKA</p>
+                            <p>{% trans "Víťa Válka" %}</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:68]:
```suggestion
                            <p>Víťa Válka</p>
```

File: weblate_web/templates/about.html [hunk 9]
```diff
@@ -47,27 +47,27 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
-                       <t>Master of the code & head of custom or large scale solutions.</t>
+                       <t>{% trans "Master of the code & head of custom or large scale solutions." %}</t>
                     </div>
                 </div>
                 <div class="weblate-team-content">
-                    <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                    <img src="https://hosted.weblate.org/avatar/80/orangesunny.png" alt="Benjamin Alan Jamie" />
                     <div class="content-p">
-                        <p>BENJAMIN ALAN JAMIE</p>
+                        <p>{% trans "Benjamin Alan Jamie" %}</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:60]:
```suggestion
                        <p>Benjamin Alan Jamie</p>
```

File: weblate_web/templates/about.html [hunk 10]
```diff
@@ -47,27 +47,27 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
             <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
             <div class="weblate-team">
                 <div class="weblate-team-content">
-                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                   <img src="https://hosted.weblate.org/avatar/80/nijel.png" alt="Michal Čihař" />
                      <div class="content-p">
-                       <p>MICHAL CIHAR</p>
+                       <p>{% trans "Michal Čihař" %}</p>
```
Comment (nijel) [inline weblate_web/templates/about.html:52]:
```suggestion
                       <p>Michal Čihař</p>
```

**Review by nijel — COMMENTED** [review]
It's not useful to localize people names.

File: weblate_web/templates/about.html [hunk 11]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="https://hosted.weblate.org/avatar/80/nijel.png" alt="Michal Čihař" />
+                     <div class="content-p">
+                       <p>{% trans "Michal Čihař" %}</p>
+                       <br>
+                       <t>{% trans "Master of the code & head of custom or large scale solutions." %}</t>
```
Comment (orangesunny) [inline weblate_web/templates/about.html:54]: Let‘s use:

*Master of the idea and code, leading the development, custom, and large scale solutions.*

File: weblate_web/templates/about.html [hunk 12]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                    </div>
+                </div>
+                <div class="weblate-team-content">
+                    <img src="https://hosted.weblate.org/avatar/80/orangesunny.png" alt="Benjamin Alan Jamie" />
+                    <div class="content-p">
+                        <p>{% trans "Benjamin Alan Jamie" %}</p>
+                        <br>
+                        <t>{% trans "Community manager & support guru helping with smooth workflows." %}</t>
```
Comment (orangesunny) [inline weblate_web/templates/about.html:62]: Let’s use:

*Community manager, support guru, helping users with smooth workflows; and the one to talk sales.*

File: weblate_web/templates/about.html [hunk 13]
```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
...
+                    </div>
+                    </div>
+                    <div class="weblate-team-content">
+                        <img src="https://hosted.weblate.org/avatar/80/vitavalka.png" alt="Víťa Válka" />
+                        <div class="content-p">
+                            <p>{% trans "Víťa Válka" %}</p>
+                            <br>
+                            <t>{% trans "Interface & graphics designer optimizing weblate user happiness." %}</t>
```
Comment (orangesunny) [inline weblate_web/templates/about.html:70]: Let’s use:

*Interface and graphics designer elevating user happiness, continually optimizing Weblate.*

**Review by orangesunny — CHANGES_REQUESTED** [review]
I rewrote the profile lines in the comments, please use them

File: weblate_web/templates/about.html (hunk 8 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Yes of course, I have updated it.

File: weblate_web/templates/about.html (hunk 11 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Done!

File: weblate_web/templates/about.html (hunk 12 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Done!

File: weblate_web/templates/about.html (hunk 13 above)
Comment (muzammalrahim) [reply weblate_web/templates/about.html]: Done!

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**